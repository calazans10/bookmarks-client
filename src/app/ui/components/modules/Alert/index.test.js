import React from 'react';
import { render } from '@testing-library/react';
import { Alert } from './index';

describe('Alert', () => {
  it('renders without crashing', () => {
    const { container } = render(
      <Alert message="Invalid credendials" isVisible onHideAlert={jest.fn()} />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
