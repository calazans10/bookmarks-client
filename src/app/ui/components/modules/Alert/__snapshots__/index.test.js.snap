// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders without crashing 1`] = `
<div
  class="sc-AykKC bjBiDs"
>
  <p
    class="sc-AykKD dfbIIS"
  >
    <span
      class="sc-AykKE ejbGjx"
    />
     
    Invalid credendials
  </p>
  <button
    aria-label="Close alert"
    class="sc-AykKF ejyTDJ"
    type="button"
  >
    <span>
      Close
    </span>
  </button>
</div>
`;
